src/SundayTasks/ClassesAndModifiers/Bed.java

```diff
@@ -0,0 +1,40 @@
+package SundayTasks.ClassesAndModifiers;
+
+public class Bed {
+    private double height;
+    private int pillows;
+    private String sheetColor;
+    private double price;
+
+    public Bed(double height, int pillows, String sheetColor, double price) {
+        this.height = height;
+        this.pillows = pillows;
+        this.sheetColor = sheetColor;
+        this.price = price;
+    }
+
+    public Bed(double height, String sheetColor) {
+        new Bed(height, ClassesAndModifiersConstants.DEFAULT_PILLOW_COUNT,
```

Comment:
```suggestion
        this(height, ClassesAndModifiersConstants.DEFAULT_PILLOW_COUNT,
```
use this to call the constructor of the current class